lib/elbas/logger.rb

```diff
@@ -4,17 +4,35 @@ module Elbas
   module Logger
     include Capistrano::Doctor::OutputHelpers
 
+    PREFIX_TEXT = '[ELBAS] '
+
     def info(message)
       $stdout.puts [prefix, message, "\n"].join
     end
 
+    def error(message)
+      $stderr.puts [error_prefix, message, "\n"].join
+    end
+
     def cyan(text)
```

Comment:
I think `cyan`,  `red`, and `color_text` should all be private

Reply:
okay, thats fine, was just following current pattern.